### PR TITLE
refactor: Enforce new sensitive inputs rules (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
image

### CODE REVIEW

File: packages/nodes-base/credentials/CiscoUmbrellaApi.credentials.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export class CiscoUmbrellaApi implements ICredentialType {
 		{
 			displayName: 'API Key',
 			name: 'apiKey',
-			// eslint-disable-next-line n8n-nodes-base/cred-class-field-unobscured-sensitive-input
+			// eslint-disable-next-line n8n-nodes-base/cred-class-field-type-options-password-missing
```
Comment (ivov) [inline packages/nodes-base/credentials/CiscoUmbrellaApi.credentials.ts:30]: Kept this as it was, but why are we exempting this?

Comment (Joffcom) [reply packages/nodes-base/credentials/CiscoUmbrellaApi.credentials.ts:30]: Good question I am not sure, I would say API keys should be impacted as well.

**Review by Joffcom — APPROVED** [review]

**Comment by ivov** [comment]
@Joffcom Closing this, I'll apply the expanded rules as discussed yesterday.

**Comment by ivov** [comment]
Superseded by https://github.com/n8n-io/n8n/pull/6815

**Pull request closed without merging**